### PR TITLE
Allow to pass options to `range-parser`

### PR DESCRIPTION
**Pull request by piranna**

### CODE REVIEW

**Comment by rexxars** [comment]
Sweet. Could you add some documentation for it in the readme?

**Comment by piranna** [comment]
> Could you add some documentation for it in the readme?

Sure, I have just added some documentation for the `range-parser` options object and the `intersectRanges` function.

File: README.md [hunk 1]
```diff
@@ -90,14 +90,14 @@ By default, up to two ranges are allowed. The limit can be adjusted with the `ma
...
 
 Optional function that allow to modify the list of ranges to be returned. This
 can be used for example if the data is not fully available because your file has
 holes in it. It will be called with an object with the same `metadata` field
 used for the `beforeSend()` optional function, and a `ranges` field with the
 list of ranges.
+
+## License
+
+MIT © [Espen Hovlandsdal](https://espen.codes/)
```
Comment (piranna) [inline README.md]: Sorry for that! :-P

**Comment by rexxars** [comment]
Thanks, released as 1.2.0!

**Comment by piranna** [comment]
Thanks to you! :-)

**Pull request merged**